hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieHiveUtils.java

```diff
@@ -68,6 +69,8 @@
   public static final String HOODIE_STOP_AT_COMPACTION_PATTERN = "hoodie.%s.ro.stop.at.compaction";
   public static final String INCREMENTAL_SCAN_MODE = "INCREMENTAL";
   public static final String SNAPSHOT_SCAN_MODE = "SNAPSHOT";
+  public static final String POINT_IN_TIME_SCAN_MODE = "POINT_IN_TIME";
```

Comment:
i think we should treat this same as snapshot. maybe we can add config hoodie.%s.consume.snapshot.time. If it is not set, use latest. What do you think?

Reply:
Ack